futurenhs.api/FutureNHS.Api/DataAccess/Database/Read/PermissionsDataProvider.cs

```diff
@@ -49,25 +49,30 @@ public Task<List<string>> GetGroupPermissionsForSiteRole(string role)
             {
                 case "Admin":
                     permissions.Add($"{Schema}/groups/{DefaultApiVerison}/edit");
+                    permissions.Add($"{Schema}/groups/{DefaultApiVerison}/leave");
```

Comment:
Is this a platform admin? I don't think we want to give platform admins the ability to leave a group @Tim-Hunt-CDS @ReallySmall ?

If they have this action it will render the 'leave group' option for platform admins

![image](https://user-images.githubusercontent.com/102808289/172837208-9e1ff28d-fa5d-40c7-9c52-b0de9e6f5dfe.png)